pkg/httpget/httpget.go

```diff
@@ -0,0 +1,36 @@
+package httpget
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+// Get sends an HTTP GET request and returns the result.
+func Get(url string) ([]byte, error) {
```

Comment:
Consider renaming the package to be something more general like httprequest? Also it is implemented similarly in urlfetcher, maybe we can consolidate them.

Reply:
Done.